PyDSS/pyControllers/Controllers/PvController.py

```diff
@@ -228,12 +247,25 @@ def VVARcontrol(self):
         """
         uMin = self.__Settings['uMin']
         uMax = self.__Settings['uMax']
-        pfLim = self.__Settings['PFlim']
         uDbMin = self.__Settings['uDbMin']
         uDbMax = self.__Settings['uDbMax']
-        Priority = self.__Settings['Priority']
 
-        uIn = max(self.__ControlledElm.sBus[0].GetVariable('puVmagAngle')[::2])
+        kVBase =self.__ControlledElm.sBus[0].GetVariable('kVBase') * 1000
+
+        Umag = self.__ControlledElm.GetVariable('VoltagesMagAng')[::2]
+        Umag = [i for i in Umag if i != 0]
+        if self.__Settings['VmeaMethod'].lower() == "mean":
```

Comment:
This means that all existing users have to add this field to their `PvController.toml` files. Each file may have a lot of entries, so it could be tedious unless people make their own scripts to add it.

This may be the proper course of action. People would have to make an explicit choice. Alternatively, you could change this code block to choose a default value if it isn't define. You could log a single warning message to alert users, perhaps.

Just raising it for your consideration. It doesn't matter for my team because we auto-generate these.